src/scripts/gen-types.js

```diff
@@ -9,15 +11,19 @@ const genTypes = async (options = {}) => {
   await svelte2tsx.emitDts({
     libRoot: basePath,
     declarationDir: outputDir,
-    svelteShimsPath: require.resolve('svelte2tsx/svelte-shims.d.ts')
+    svelteShimsPath: path.resolve('svelte2tsx/svelte-shims.d.ts')
   })
 }
 
-module.exports = genTypes
+export default genTypes
 
-if (require.main == module) {
-  genTypes({
-    basePath: './',
-    outputDir: './types'
-  }).then(() => console.log('Done'))
+// Check if this module is being run directly
+if (import.meta.url.startsWith('file:')) {
+  const modulePath = fileURLToPath(import.meta.url)
+  if (process.argv[1] === modulePath) {
```

Comment:
maybe this needs to be a bit more robust? I wonder if we should extract it to a common function